app/views/shared/_side-nav.html.erb

```diff
@@ -8,18 +8,26 @@
     <ul class="main-menu">
         <li class="main-menu__item">
             <%= link_to_unless_current "Needs", root_path, class: "main-menu__link" do %>
```

Comment:
Is this line necessary now that we have a `link_to` in the block?

Reply:
Yes, It is necessary.  When we are 'in' submenu option we want to see `Needs` selected and also to be able to press the link 'Needs'.